abis_mapping/types/spatial.py

```diff
@@ -100,14 +99,21 @@ def original_datum_uri(self) -> rdflib.URIRef | None:
         """Getter for the original datum URI.
 
         Returns:
-            rdflib.URIRef: Uri corresponding to original datum if known, else None.
+            rdflib.URIRef: Uri corresponding to original datum.
```

Comment:
update function return type to just `rdflib.URIRef`

Reply:
thinking you meant change the function return *annotation*? changed.